lease/lessor.go

```diff
@@ -22,6 +22,7 @@ import (
 	"sync"
 	"time"
 
+	"github.com/ScaleFT/monotime"
```

Comment:
let's put a space between external dependency and internal dependency (coreos/etcd/xxx)